client/my-sites/earn/main.jsx

```diff
@@ -185,54 +177,32 @@ class EarningsMain extends Component {
 		);
 	};
 
-	render() {
-		const { adsProgramName, section, translate } = this.props;
-		const component = this.getComponent( this.props.section );
-
-		const layoutTitles = {
-			earnings: translate( '%(wordads)s Earnings', { args: { wordads: adsProgramName } } ),
-			settings: translate( '%(wordads)s Settings', { args: { wordads: adsProgramName } } ),
-			payments: translate( 'Recurring Payments' ),
-			'payments-plans': translate( 'Recurring Payments plans' ),
-			'refer-a-friend': translate( 'Refer-a-Friend Program' ),
-		};
-
-		return (
-			<Main wideLayout={ true } className="earn">
-				<PageViewTracker
-					path={ section ? `/earn/${ section }/:site` : `/earn/:site` }
-					title={ `${ adsProgramName } ${ capitalize( section ) }` }
-				/>
-				<DocumentHead title={ layoutTitles[ section ] } />
-				<FormattedHeader
-					brandFont
-					className="earn__page-header"
-					headerText={ translate( 'Earn' ) }
-					subHeaderText={ translate(
-						'Explore tools to earn money with your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
-						{
-							components: {
-								learnMoreLink: <InlineSupportLink supportContext="earn" showIcon={ false } />,
-							},
-						}
-					) }
-					align="left"
-				/>
-				{ this.getHeaderCake() }
-				{ section && this.getSectionNav( section ) }
-				{ component }
-			</Main>
-		);
-	}
-}
-
-export default connect( ( state ) => {
-	const siteId = getSelectedSiteId( state );
-
-	return {
-		canAccessAds: canAccessWordAds( state, siteId ),
-		site: getSelectedSite( state ),
-		siteId,
-		siteSlug: getSelectedSiteSlug( state ),
-	};
-} )( localize( EarningsMain ) );
+	return (
+		<Main wideLayout={ true } className="earn">
+			<PageViewTracker
+				path={ section ? `/earn/${ section }/:site` : `/earn/:site` }
+				title={ `${ adsProgramName } ${ capitalize( section ) }` }
+			/>
+			<DocumentHead title={ layoutTitles[ section ] } />
+			<FormattedHeader
+				brandFont
+				className="earn__page-header"
+				headerText={ translate( 'Earn' ) }
+				subHeaderText={ translate(
+					'Explore tools to earn money with your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+					{
+						components: {
+							learnMoreLink: <InlineSupportLink supportContext="earn" showIcon={ false } />,
+						},
+					}
+				) }
+				align="left"
+			/>
+			{ getHeaderCake() }
+			{ section && getSectionNav( section ) }
+			{ getComponent( section ) }
+		</Main>
+	);
+};
+
```

Comment:
For some reason, Github is not showing detailed changes for the markup being returned for this component (ie, everything inside <Main>). But changes here are minimal. The render() function has been removed and class syntax has been updated (ie, no this.method() or this.props anymore).

Reply:
Hiding the whitespace changes helps:

<img width="255" alt="image" src="https://github.com/Automattic/wp-calypso/assets/87168/119ed7c2-73fe-479e-9a3e-ce127fc717d6">